test/compiler/js/style.js

```diff
@@ -1,7 +1,6 @@
 riot.tag('style-test', '<h3>{ opts.title }</h3>', 'style-test { display: block; border: 2px }', function(opts) {
-    this.foo = function() {
-
-        this.update()
-    }.bind(this);
+  this.foo = function() {
```

Comment:
The [guidelines](https://github.com/muut/riotjs/blob/master/CONTRIBUTING.md) has been there for a week or so. I want to keep it small so that people actually read it and remember it. Please tell me what is missing from there!